### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jonathanstanley**
trellis into its own directory and clarify install process

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,9 +19,10 @@ This project can be cloned and re-configured to fit your needs but we highly sug
 Here's how this example project was created:
 
 1. Create a new project directory: `$ mkdir example.com && cd example.com`
-2. Clone Trellis: `$ git clone --depth=1 git@github.com:roots/trellis.git && rm -rf trellis/.git`
+2. Clone Trellis: `$ git clone --depth=1 git@github.com:roots/trellis.git trellis && rm -rf trellis/.git`
 3. Clone Bedrock: `$ git clone --depth=1 git@github.com:roots/bedrock.git site && rm -rf site/.git`
 4. Clone Sage: `$ git clone --depth=1 git@github.com:roots/sage.git site/web/app/themes/sage && rm -rf site/web/app/themes/sage/.git`
+5. From Bedrock directory, run `composer install`
```
Comment (swalkinshaw) [inline README.md:25]: First change is good but this one isn't necessary. Trellis automatically runs `composer install` for you during development.

File: README.md [hunk 2]
```diff
@@ -19,9 +19,10 @@ This project can be cloned and re-configured to fit your needs but we highly sug
 Here's how this example project was created:
 
 1. Create a new project directory: `$ mkdir example.com && cd example.com`
-2. Clone Trellis: `$ git clone --depth=1 git@github.com:roots/trellis.git && rm -rf trellis/.git`
+2. Clone Trellis: `$ git clone --depth=1 git@github.com:roots/trellis.git trellis && rm -rf trellis/.git`
```
Comment (retlehs) [inline README.md:22]: this was done on purpose [when we renamed dirs](https://github.com/roots/roots-example-project.com/commit/f108d88037b590b18c1072207e0afc43b750c7e2#diff-04c6e90faac2675aa89e2176d2eec7d8R22) — trellis is already going to be the name of the directory upon running this command

**Comment by swalkinshaw** [comment]
See https://github.com/roots/roots-example-project.com/pull/34#discussion_r45828259

**Comment by jonathanstanley** [comment]
No problem.  In my experience it laborious to figure all that out and thought I'd try to help clarify for others as they get initiated to roots

**Pull request closed without merging**